lib/access_import/access_import.rb

```diff
@@ -20,8 +20,17 @@ def make_clients
       ).any?
       client_attrs = client_transformer.prepare_attributes(ac_client)
       client = Client.new(client_attrs)
+      client.created_at = ac_client[:d_Rollenbeginn]
+      client.updated_at = ac_client[:d_MutDatum]
       client.user = User.first
-      puts "Importing personen_rolle #{key} to Client.id: #{client.id}" if client.save!
+      next unless client.save!
+      message = "Importing personen_rolle #{key} to Client.id: #{client.id}"
+      if ac_client[:d_Rollenende]
+        client.state = Client::FINISHED
+        client.save!
```

Comment:
In this case the `client` gets saved twice. Can you reduce this to only one `save` statement? Probably using `valid?` for the above statement and moving `save` after this `if` statement.